CHANGELOG.md

```diff
@@ -11,6 +11,26 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-3807**
   - Pinned @aws-sdk/client-s3 to 3.614 to address timeout/bug in s3().listObjectsV2
 
+### Added
+
+- **CUMULUS-3760**
+  - Added guidance for handling large backlog of es executions
+- **CUMULUS-3742**
```

Comment:
@paulpilone this was your original question which made me actually include the changes from 3742 and update the CL.